### DISCUSSION TITLE
Localnet RDP brute 192.168.1/24 is not blocked

### DISCUSSION THREAD

**Discussion by alexandertischenko** [DigitalRuby/IPBan#152] [Q&A] [answered]
I've use IPBan for block RDP bruteforce on Windows 2016 Server in VMWare environment for testing purposes. IPBan doesn't work (doesnt block 10 failure logins with default settings) until i've change local subnet from 192.168/24 to 5.5/24. After changing local network addresses (attacker VM host 5.5.5.100 and ipban featured VM server 5.5.5.10) works fine. I haven't find any exclusions for private IP address ranges in config files. Is this feature of IPBan or bug?

**Comment by jjxtra** [comment, marked as answer]
Internal ips are ignored currently.
